api/envoy/service/discovery/v3/discovery.proto

```diff
@@ -274,12 +274,18 @@ message Resource {
   // The resource being tracked.
   google.protobuf.Any resource = 2;
 
-  // Time-to-live value for the resource. For each resource, a timer is started. The timer is reset each time the
-  // resource is received with a new TTL. If the resource is received with no TTL set, the timer is removed for the
-  // resource. Upon expiration of the timer, the configuration for the resource will be removed.
+  // Time-to-live value for the resource. For each resource, a timer is started. The timer is
+  // reset each time the resource is received with a new TTL. If the resource is received with
+  // no TTL set, the timer is removed for the resource. Upon expiration of the timer, the
+  // configuration for the resource will be removed.
   //
-  // The TTL feature is meant to support configurations that should be removed in the event of a management server
-  // failure. For example, the feature may be used for fault injection testing where the fault injection should be
-  // terminated in the event that Envoy loses contact with the management server.
+  // The TTL can be refreshed or changed by sending a response that doesn't change the resource
+  // version. In this case the resource field does not need to be populated, which allows for
+  // light-weight "heartbeat" updates to keep a resource with a TTL alive.
+  //
+  // The TTL feature is meant to support configurations that should be removed in the event of
+  // a management server // failure. For example, the feature may be used for fault injection
```

Comment:
typo here, double "//"

Reply:
Yea will fix in follow up just to land this, thanks!